docker-images/run-pg-docker.sh

```diff
@@ -57,11 +57,29 @@ if [ ! -f "Dockerfile" ]; then
     exit 4
 fi
 
-# now build the container
-DOCKER_CONTAINER_NAME=${DOCKER_IMAGE_TO_RUN}_learn_postgresql_1
-$SUDO $DOCKER_COMPOSE build --force-rm --no-cache
+# Check if image exists locally
+DOCKER_IMAGE_NAME=$(docker-compose config | grep 'image:' | awk '{print $2}' | head -n 1)
+IMAGE_EXISTS=$(docker images -q "$DOCKER_IMAGE_NAME")
+
+if [ -z "$IMAGE_EXISTS" ]; then
+    # If the image doesn't exist, build it
+    echo "Building the Docker image $DOCKER_IMAGE_NAME..."
+    $SUDO $DOCKER_COMPOSE build --force-rm --no-cache
```

Comment:
There is a meaning for this: we don't want inexperienced user to waste time with previously misconfigured instances, so the script was always rebuilding it from scratch.
While I agree that this is often unnecessary, it could help fixing some problems when the user fires up the same instance over and over without having an idea of what changes he made permanently in the previous run.
Can we make this an option?